plugins/woocommerce/tests/e2e-pw/tests/merchant/customer-list.spec.js

```diff
@@ -78,6 +78,16 @@ baseTest.describe( 'Merchant > Customer List', () => {
 		},
 	} );
 
+	test.beforeAll( async ( { api } ) => {
+		let oldCustomers = {};
+		await api.get( 'customers' ).then( ( response ) => {
+			oldCustomers = response.data;
+		} );
+		await api.post( `customers/batch`, {
```

Comment:
Will it create an api error if `oldCustomers` is empty?

Reply:
Nope, I tested it locally on my machine where there are no customers and no error is thrown.  So it works either way.